ERP_INES/Data/Modules/Finance/Repositories/Interfaces/ITransactionRepository.cs

```diff
@@ -4,7 +4,12 @@ namespace ERP_INES.Data.Modules.Finance.Repositories.Interfaces;
 
 public interface ITransactionRepository
 {
-    Task<List<Transaction>> GetTransactionsAsync(string? currency);
+    Task<List<Transaction>> GetTransactionsAsync(string? currency,
+        string? sort,
+        string order="asc",
+        int page = 1,
+        int limit = 1000
```

Comment:
#would: don't you think that 1000 is to much?
I would like to suggest just 50 at maximum, and of course we could provide some endpoint to grab all and export to csv or something.